src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java

```diff
@@ -133,13 +147,13 @@ public ResponseEntity<Page<ArticleListDTO>> getAllArticles(Pageable pageable) {
             articleService.getAllArticles(pageable));
     }
 
-    @PreAuthorize("hasAnyAuthority('ADMIN')")
-    @GetMapping("/admin/articles/category_id/{id}")
+    @PreAuthorize("hasAnyAuthority('USER','ADMIN')")
```

Comment:
return @PreAuthorize("hasAnyAuthority('ADMIN')") because this method is only for user